### ISSUE TITLE
Nouvelles maquettes à prévoir

### ISSUE THREAD

**Issue by bzg** [Jardin-des-Sciences/website#35]
(En référence à https://github.com/Jardin-des-Sciences/website/issues/24)

Un point **très important** - @Issenmann pour validation.

Il est clair que _toutes les pages n'ont pas été maquettées_, et qu'il faudra une page dédiée par type de contenu : une page présentant un visuel, une page présentant un sondage, etc.

Je propose qu'on chiffre deux maquettes supplémentaires, chacune servant de matrice pour plusieurs contenus:

- une maquette "média" pour les images, les vidéos, les diaporamas, les sons ;
- une maquette "autres contenus" pour les événements, les sondages, les QCM, les questions éditorialisées, les entrées de glossaire, les FAQ, les liens.

Dis-nous si l'approche te semble raisonnable.

S'il faut sacrifier un type de contenu, nous pouvons en parler.

**Comment by bzg** [comment]
Note: nous aurons peut-être à faire une maquette distincte pour l'édition d'événements.

**Comment by Nicoloye** [comment]
Le point est noté de notre côté, nous demanderont à Béa de chiffrer le point et je livrerai ce chiffrage dans une ligne à part dans la livraison du chiffrage prévue courant de semaine prochaine.

**Comment by Issenmann** [comment]
oui, ok pour les deux maquettes contenus et éventuellement maquette événements. à voir si la seconde peut englober tous les types de contenus.
